public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -2255,7 +2272,7 @@ function register_post_types() {
 			array(
 				'labels'          => $labels,
 				'rewrite'         => array( 'slug' => 'organizer', 'with_front' => false, ),
-				'supports'        => array( 'title', 'editor', 'revisions' ),
+				'supports'        => array( 'title', 'editor', 'excerpt', 'revisions' ),
```

Comment:
`MIGHT` - I'd remove this as well, and anything else related to Organizers, since it'll make more sense to do it all in a single commit (or maybe 1 for the REST API stuff and 1 for the block itself)

Reply:
Keeping this for now because its a bit messy to separate out changes in a single file over various commits.

Reply:
👍